opencti-platform/opencti-front/src/private/components/workspaces/workspaceExportHandler.ts

```diff
@@ -0,0 +1,27 @@
+import fileDownload from 'js-file-download';
+import pjson from '../../../../package.json';
```

Comment:
So openCTI_version is the version we have in package.json of opencti-front ?

Reply:
Yes the one from the export platform